### DISCUSSION TITLE
《原子習慣》閱讀筆記

### DISCUSSION THREAD

**Discussion by JohnnyXiao1024** [1101-GS4719-C/Python-Community-GS4719-C#1206] [General]
## 前言
各位午安，連假除了好好衝刺期中考
感覺也是個償還書債的好時候，我也堅信筆記的撰寫有助於個人對文本的吸收
這本《原子習慣》相信大家都不陌生，以下是我這幾天閱讀後的小小摘要
學校圖書館有電子資源，非常推薦各位花個一兩天仔細閱讀喔～

### 一、讓「提示」顯而易見：

1. 行為的改變始於「察覺」

有足夠的練習，就能不假思索的看出預示某種結果的提示

（如 經驗充足的醫師可以看出某個路人的臉色怪怪的）

習慣只要自動化，就難以察覺

若想改變行為，就需意識到自己在做的事

指差確認＞讓「無意識」的習慣變為「有意識」

填寫習慣記分卡，讓自己得以察覺

2. 讓「提示」顯而易見

執行意向「於何時 在何地 做何事」

習慣堆疊 「舊習慣完成後，做新習慣」

越明確越好，時間和地點是常用提示

3. 環境的重要性

習慣是由「提示」驅動的，我們容易注意到更突出的提示

讓好習慣的提示充滿環境，久而久之，好習慣不只與單一提示連結，而是與整個情境結合，換句話說情境成了提示（如 在床上只睡覺、在書桌只讀書）

在新環境養成新習慣較簡單，因為不用抵抗舊習慣

### 二、讓習慣有「吸引力」：

1. 吸引力越大，越可能形成習慣

習慣是多巴安驅動的回饋迴路，多巴安升高，行為的動機也隨之提升（尻槍）

讓人採取行動的，是對獎賞的「預期」，而非「實現」，期待越高，多巴安峰值越高

誘惑綑綁：將想要與需要綑綁

2. 歸屬：家人、朋友與文化

模仿親近的人（社會環境）：加入文化，並滿足兩個特點：在該文化中，你想要的行為是常態＋你跟這個群本就有些共同點

模仿多數的人：從眾效應

模仿有力的人：若一個行為能帶來尊敬和讚美，其吸引力便會增加

3. 創造動機儀式

找一件真正讓你快樂的事，創造一個簡短儀式在做快樂的事之前執行（如 你喜歡摸狗狗，創造一個深呼吸三次，然後微笑的簡短儀式〉〉深呼吸三次，微笑，摸狗狗，重複。

久而久之，「深呼吸三次，微笑」就成了帶來快樂的提示，即使它已脫離原來的情境。

### 三、讓行動輕而易舉

1. 由重複開始，而非完美

一直處於啟動狀態而不行動：想延後失敗

最有效的學習是實行，而非計畫

學習曲線：習慣的改變取決於頻率，而非時間〉多試幾次，跨越習慣線〉自動化

2. 最小努力原則

並非只做簡單的事，而是讓「長期做下來會有回報的事」變得輕而易舉

創造出一個讓「目標事項」可能輕而易舉實行的環境，比起「拿出手機〉打開軟體〉搜尋」，直接喊一聲「播放音樂」更為省力

減少與好習慣間的阻力（前一晚準備好鍋子、食材）

增加與壞習慣間的阻力（把電視插頭拔掉，遙控器拔電池）

很多個小改變累積起來，會有很大的效益

3. 兩分鐘法則

前期幾秒鐘內的決定，可能影響幾分鐘、幾小時後的行為（出門吃飯＞去牛排館＞只能點菜單上的）

新習慣的開始應該要花不到兩分鐘（每週健身三次＞每週去一次＋一段時間＞每週去一次＋五分鐘就好＞穿鞋出門＞穿鞋）

還記得身份認同嗎，即使每天只去健身房五分鐘，但只要連續去五天，你就是把票投給你想建立的新身份，重點不只是練出好身材，而是成為不能錯過健身的那種人

做一些，好過什麼都不做，先標準化，才能最佳化。我們無法改善不存在的習慣

4. 一次性選擇：買更好的床墊或註冊自動儲蓄計劃
5. 利用科技讓習慣自動化：好習慣不是可能發生，而是必然

### 四、讓獎賞令人滿足

1. 讓大腦知道：「這感覺很好，下次還要做」

立即獎賞的行為會被重複，反之亦然

立即獎賞＞未來獎賞：帶來立即獎賞的行為會被重複

為長遠有益的習慣添加立即的愉悅，反之亦然

抵抗誘惑（略過酒吧特價）沒什麼好令人滿足的

讓避免變得可見：每略過一次亂買，轉錢進去終極目標的帳戶裡

但 短期獎賞和最終目標不能牴觸（如 健身完後吃冰淇淋）

兩者需投給相同的身份認同 （如 健身完後去按摩）

獎賞的滿足，提高我們重複的可能性

重視「成功」的感受，即使規模很小

2. 每天堅持好習慣

習慣追蹤：利用視覺測量，讓堅持的成果可見，讓自己撒不了謊（如 日期打勾、數據）

「不要中斷紀錄」：專注於過程，·不是結果（不是練出六塊肌，而是成為不會錯過健身的人）

「不貳過」：只能錯一次

難處 同時養成兩個習慣：節食＋紀錄食物

習慣堆疊＋習慣追蹤（做完目前習慣，我會追蹤習慣）＞當下紀錄

非必要，別中斷：不在於健身發生的事，而在於成為不會錯過健身的那種人

健身房五分鐘雖無法改善體能，卻能強化身份認同

古德哈特定律：「當測量成了目標，就不再是個好的測量方法」

（例：追求超多下胸推，動作卻不確實）

「非量化成就」：減重 可能體重機上的數字不變，卻讓你皮膚變好、作息規律等

### 一、讓「提示」消失：

1. 自制力的真相

迴避誘惑比拒絕誘惑簡單，有高度自制力的人不常待在充滿誘惑的環境裡

要剷除壞習慣，就減少接觸其提示

自制力是短期策略，撐不久

### 二、讓習慣喪失「吸引力」：

表層的渴望與深層的動機 （如 經營社交軟體＝＞被認同），換句話說，古老慾望的現代解決方案

強調避免某個壞習慣後，可能得到的益處，使得壞習慣失去吸引力

和正面感受連結，可提高習慣吸引力，反之亦然（如 被限制在輪椅上；透過輪椅離開病床）

### 三、讓行動困難到爆

1. 承諾機制：利用當下的選擇導向未來比較好的行為
2. 提高阻力：增加與壞習慣間的步驟

### 四、讓後果令人不滿

1. 問責夥伴的重要

習慣契約：加快與惡習相關懲罰的到來速度（如果做不到，就....)

問責夥伴能為「不作為」創造立即的代價，因為我們在乎別人怎麼看自己

## 更上一層樓

1. 基因：選對戰場

基因不會消除努力的重要性，而是讓努力的方向清晰，告訴自己要努力去做「什麼」

選擇對自身有利的戰場，若找不到，就自己創造一個

2. 金髮女孩原則：剛剛好的難度

成功最大的威脅不是失敗，而是無聊

專業人士堅守既定行程，業餘者接受生活干擾

最有價值的是，那天興趣缺缺依然堅持做完

3. 建立好習慣的壞處

難以注意到微小失誤＞反省與複查

精通＞掌握一項習慣後，以此為基礎繼續精進，將空出來的精力投入下一個習慣的建立中

（習慣＋刻意練習＝精通）

越是抓緊一個身份認同不放，越難超越他

我是軍人＞我是那種自律、可靠且擅長團隊合作之人

我是運動員＞我是那種心裡強悍且熱愛體能挑戰之人

## 結語

如果難以記住某習慣「讓提示顯而易見」

有時缺乏起頭的動力「讓習慣有吸引力」

執行某習慣太困難「讓行動輕而易舉」

不想堅持下去「獎賞令人滿足」

**Comment by JohnnyXiao1024** [comment]
btw 市面上其實有著許多關於建立習慣、培養自律的書籍
近年甚至也出了許多如何讀書、如何閱讀、如何背單字的工具書
但比起閱讀一堆工具書籍，我認為最重要的還是能夠實際應用他們
如果花了一堆時間學習各式工具卻不應用，也就太可惜了.....

**Comment by DuxtinTai** [comment]
整理的很清楚!現在真的很多這種工具書，原子習慣算是很不錯的!
我也來分享一些讀後感，書中介紹了如何設定目標、如何保持動力和如何克服挫折，並將目標轉化為具體的行動步驟，以及如何將它們納入到日常生活中。其中我覺得最重要的是，要培養一個習慣最直接的方式是將自己想像成擁有這個習慣的人，也就是從改變對自己的認知開始。

**Comment by brian431** [comment]
感謝您整理的筆記，要點都整理的十分明瞭。養成好習慣跟戒除壞習慣確實是非常重要的事情，甚至可能影響一輩子。不過做起來卻沒有說起來的這麼容易。尤其對我這種個性隨隨便便的人來說，要下定決心去做一些事情真的挺困難的。不過這本書倒是提供了些不錯的方法，非常值得參考。

**Comment by zilinng** [comment]
我也有看過這本書，其中提到的cue點對我來說很有幫助，把想要養成的習慣和平常本來就會做的事情綁在一起，後期真的可以不知不覺的開始做本來很難開始的事情，我是把吹頭髮當作cue點，吹完頭髮就要開始看書，那段時間真的有成功養成持續閱讀的習慣。

**Comment by tower20021202** [comment]
我之前也有看過這本書，其中有幾點像是把目標放小一點這樣很多時候更能自然而然地去執行事情，像是我現在都把走進健身房當作目標而不是要做多少組動作，這樣更能讓我習慣上健身房這件是，動做也會因上健身房而自然的就完成了

**Comment by zephyrxin01** [comment]
我之前看完這本書以後，很認真執行了一陣子。但同樣隨著一環沒有做到以後就索性放棄了。習慣的養成真的非常不容易，環境的改變和突如其來的忙碌行程都很容易打斷習慣的養成。但我認為這些要點的確是有他的合理性和可執行性的。

**Comment by harrycoconut** [comment]
這本書我非常喜歡！原子習慣中有提到一個非常好的觀點，讓提示顯而易見，我也透過這個方法建立起了每天彈吉他的習慣😌

**Comment by hardtagottaname** [comment]
我記得之前有看過一個類似的概念，內容是說要持續一個習慣至少30天，讓你的大腦能夠習慣，變成日常必做事情之一。因此，希望我可以養成好習慣，比如不要賴床、能夠每天固定一段時間用在讀書，並且杜絕掉一些會讓我日漸墮落的壞習慣，例如縮短看網路小說、追網劇等等浪費時間的行為。
